examples/remix-auth-auth0/app/root.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration
+} from "remix";
+
+export default function App() {
+  return (
+    <html lang="en">
+      <head>
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        <Meta />
+        <Links />
+      </head>
+      <body>
+        <Outlet />
+        <ScrollRestoration />
+        <Scripts />
+        {process.env.NODE_ENV === "development" && <LiveReload />}
```

Comment:
```suggestion
        <LiveReload />
```
See https://github.com/remix-run/remix/pull/1352

Reply:
Thank you!
I fixed it.